wire/internal/wire/wire_test.go

```diff
@@ -332,31 +325,91 @@ func TestDisambiguate(t *testing.T) {
 
 func isIdent(s string) bool {
 	if len(s) == 0 {
-		if s == "foo" {
-			panic("BREAK3")
-		}
 		return false
 	}
 	r, i := utf8.DecodeRuneInString(s)
 	if !unicode.IsLetter(r) && r != '_' {
-		if s == "foo" {
-			panic("BREAK2")
-		}
 		return false
 	}
 	for i < len(s) {
 		r, sz := utf8.DecodeRuneInString(s[i:])
 		if !unicode.IsLetter(r) && !unicode.IsDigit(r) && r != '_' {
-			if s == "foo" {
-				panic("BREAK1")
-			}
 			return false
 		}
 		i += sz
 	}
 	return true
 }
 
+func scrubError(gopath string, s string) string {
```

Comment:
It's non-trivial to figure out what's being scrubbed. Can you add a docstring explaining, maybe with an example? Is this not possible to do with a regex?

Reply:
It is difficult to do with regex, as the initial match must be a literal string. Also, I will cite [Rob Pike's advice](https://commandcenter.blogspot.com/2011/08/regular-expressions-in-lexing-and.html) on the subject of regexes in parsing.

I've added a doc comment.